.dockerignore

```diff
@@ -1,6 +1,6 @@
 .git
 lnd/.git
 configurator
-!configurator/target/aarch64-unknown-linux-musl/release/configurator
+!configurator/target/
```

Comment:
This nullifies the benefits of excluding the configurator binary. The point of this line is to ignore everything in `target` *except* the configurator binary. By adding `configurator/target` as an exception to the exclusion, you have added back everything in `target`, which when included, is the primary reason the builds take so long.

Reply:
fixed